File: backEstacionamento/Models/Modelo.cs
using System.ComponentModel.DataAnnotations;

public class Modelo
{
    [Key]
    public int? _idModelo {get;set;}
    public string? _nomeModelo {get; set;}
    public int? _AnoModelo{get;set;}
    public String? _TipoModelo {get; set;}
    public int? _idMarca{get;set;}
 
    //-------------------------------------------------------------------------
    public Marca? Marca{get;set;}

    //-------------------------------------------------------------------------
    
   
    
}

File: frontEstacionamento/src/app/Modelo.ts
import { Marca } from "./Marca";

export class Modelo {
    _idModelo: number = 0;
    _nomeModelo: String = "";
    _motor: String = "";
    _qtdPortas: Number = 0;
    _AnoModelo: number = 0;
    _TipoModelo: String = "";
    _idMarca: Marca | undefined;
}

